src/sagemaker/jumpstart/notebook_utils.py

```diff
@@ -393,7 +393,9 @@ def _generate_jumpstart_model_versions(  # pylint: disable=redefined-builtin
     if isinstance(filter, str):
         filter = Identity(filter)
 
-    manifest_keys = set(models_manifest_list[0].__slots__)
+    manifest_keys = set(
```

Comment:
that's a bit concerning that this does not cause any unit tests to fail

Reply:
It's because these keys are exactly the same. I was thinking to add a hypothetical new key in a test, but then I feel it should be done when we actually change the slots in `JumpStartModelHeader`. Let me know if you feel strongly to add it now

Reply:
so you're say it happens to work now, cause they're the same. i guess that's fine. what about the spec keys though?